### PR TITLE
Fixed border radius of project card

### LINKED ISSUE #3631: [UI] Card radius is not correctly showing
#### Description
In the given screenshot you can see card radius is not correctly showing

#### Page Link:
https://layer5.io/projects

#### Expected Behavior
I made changes via browser console and It should look like something that screenshot that I provided below

#### Screenshots

**Current**

Screenshot 2023-01-17 at 10 52 16 PM

**Expected**

Screenshot 2023-01-17 at 10 36 03 PM

#### Environment:
- Host OS: 
- Browser: 

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Yashsharma1911**
Signed-off-by: Yash sharma <71271069+Yashsharma1911@users.noreply.github.com>

**Description**
Changed border-radius of project__card from % unit to px unit to get a correct curved border in display resolution

This PR fixes #3631 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Projects/Project-grid/projectGrid.style.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ export const ProjectWrapper = styled.div`
     .project__card{
         box-shadow: 0px 0px 10px ${props => props.theme.DarkTheme ? "#00d3a9" : "lightgrey"};
         background-color: ${props => props.theme.DarkTheme ? "#212121" : "#FFFFFF"};
-        border-radius: 4%;
+        border-radius: 11px;
```
Comment (Nikhil-Ladha) [inline src/sections/Projects/Project-grid/projectGrid.style.js:54]: Update to `0.5rem` instead

**Comment by l5io** [comment]
🚀 Preview for commit d7faf49145cec08eebec551cff3fb9b2a1a6bec8 at: https://63ccefad4399b9621e8a386a--layer5.netlify.app

**Comment by Yashsharma1911** [comment]
Hey @Nikhil-Ladha I have made the change

**Comment by l5io** [comment]
🚀 Preview for commit cbc8724e23b2441fbaef6adbe1ba91cd189042c5 at: https://63cd0200434a957394ca25c3--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**